skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -497,6 +498,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Sort/-OrderBy
```

Comment:
We can't use more than one parameter name here, please split in two and also add the yaml section to conform with cmdlet's article format

Reply:
@get-itips I've modified it. Thanks!